Makefile.dune

```diff
@@ -37,6 +41,20 @@ release: voboot
 apidoc: voboot
 	dune build $(DUNEOPT) @doc
 
+DUNE_BOOT_VER=1.3.0
+DUNE_TBZ=dune-$(DUNE_BOOT_VER).tbz
+DUNE_BOOT_BIN=_dune/dune-$(DUNE_BOOT_VER)/_build/install/default/bin/dune
+bootstrap:
+	mkdir -p _dune
+	( cd _dune && wget https://github.com/ocaml/dune/releases/download/$(DUNE_BOOT_VER)/$(DUNE_TBZ) && tar xvfj $(DUNE_TBZ) )
```

Comment:
It is a bit weird to have a `wget` as part of a Makefile. Will it be OK with e.g. Debian policies? In any case, maybe this download step should be split up to a separate target so that package maintainers are allowed to do this step in their own pre-install script and not have the Makefile do this.

Reply:
It would be a violation of the Debian policy if we required to use the `wget` target to build the package, but indeed, in Debian the build would depend on Dune and use the system one so this is OK.